### PR TITLE
Adds notes

### PR DESCRIPTION
**Pull request by Galithil**
adds notes to worksets. Only for storage purposes.

### CODE REVIEW

File: scilifelab/lims_utils/workset_upload.py [hunk 1]
```diff
@@ -88,6 +89,13 @@ def __init__(self, lims, crawler, log):
         self.obj['technician']=crawler.starting_proc.technician.initials
         self.obj['id']=crawler.starting_proc.id
         self.obj['date_run']=crawler.starting_proc.date_run
+        self.obj['workset_notes']={}
+        if "Workset Notes" in crawler.starting_proc.udf:
+            notes=json.loads(crawler.starting_proc.udf["Workset Notes"])
+            for date, note in notes:
```
Comment (guillermo-carrasco) [inline scilifelab/lims_utils/workset_upload.py]: hmm what is this `for` for?

Comment (Galithil) [inline scilifelab/lims_utils/workset_upload.py]: ha, it's an artefact of the time where I wanted to store the notes in statusdb. sorry

**Comment by guillermo-carrasco** [comment]
hmm now you removed the workset notes as well, is that what you wanted? So you basically are just adding Sverker as responsible, right? (and importing `json` :stuck_out_tongue: )

**Comment by Galithil** [comment]
yes. the import json is not needed though. I'll remove it

**Pull request merged**